tests/clds_hash_table_int/clds_hash_table_int.c

```diff
@@ -23,6 +23,8 @@
 
 static TEST_MUTEX_HANDLE test_serialize_mutex;
 
+#define XTEST_FUNCTION(A) void A(void)
+
```

Comment:
do you need this any more?

Reply:
Nope, removed, was a debugging left over.